components/automate-ui/src/app/entities/users/user.actions.ts

```diff
@@ -130,11 +154,15 @@ export type UserActions =
   | GetUser
   | GetUserSuccess
   | GetUserFailure
-  | UpdateUser
-  | UpdateUserSuccess
+  | UpdatePasswordUser
```

Comment:
Broke out the update password and update name to allow for different type of notifications of success. The update password sends a notification to the banner and the update name displays an notification next to the save button.